app/controllers/stories_controller.rb

```diff
@@ -330,4 +331,58 @@ def assign_podcasts
   def assign_classified_listings
     @classified_listings = ClassifiedListing.where(published: true).select(:title, :category, :slug, :bumped_at)
   end
+
+  def set_json_ld
+    @user_json_ld = {
+      "@context": "http://schema.org",
+      "@type": "Person",
+      "mainEntityOfPage": {
+        "@type": "WebPage",
+        "@id": "<%= user_url(@user) %>"
```

Comment:
Since now we're writing Ruby, this won't be interpreted by the ERB (the template) parser, so it'll just be a string with tha value, see: 

![Screenshot_2020-04-15 Structured Data Testing Tool](https://user-images.githubusercontent.com/146201/79337300-8760b180-7f25-11ea-9708-c057cc931877.png)

this for example has to become: `"@id": URL.user(@user)`